frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts

```diff
@@ -92,7 +95,12 @@ Cypress.Commands.add('install', (mode: 'Internal' | 'Attached' = 'Internal', enc
             expect(item.hasOwnProperty('cluster.ocs.openshift.io/openshift-storage')).toBeTruthy(),
           );
       });
-
+      // check if taint is applied
+      cy.log('verifying taint is applied to worker nodes');
+      cy.exec('oc get nodes -l "node-role.kubernetes.io/worker" -o json').then((res) => {
+        const { items } = JSON.parse(res.stdout);
+        items.forEach((item) => expect(item.spec.hasOwnProperty('taints')).toBeTruthy());
```

Comment:
Let's test that it has the particular taint that we are looking for or not. It should be defined as OCS taint.